public/css/widget/state-change.less

```diff
@@ -2,7 +2,8 @@
   display: inline-flex;
 
   .state-ball {
-    box-shadow: 0 0 0 1px @body-bg-color;
+    box-shadow: 0 0 0 1px;
+    .var(color, body-bg-color); // box shadow color
```

Comment:
Apart from this change, everything looks fine. Because that's not how it's going to work. You should either define a variable for it or leave it as it was before.

Reply:
I got this idea from [SO](https://stackoverflow.com/questions/3012899/is-there-a-box-shadow-color-property). @nilmerg  What do you think about it? I will test it again on all the browsers we support.

Reply:
In Icinga Web 2 `box-shadow` was already a subject for such changes. Take a look there how it's been solved.